src/lib.rs

```diff
@@ -27,7 +28,7 @@ pub use server::LspServer;
 
 #[macro_export]
 macro_rules! walk_ast_package {
-    ($visitor:expr, $package:ident) => {{
+    ($visitor:expr, $package:expr) => {{
```

Comment:
Why this change? The two uses are identifiers, and it checks when I change it back to ident. What am I missing?

Reply:
Hm, I don't remember. I think I went to use it a different way, and then ended up not needing it. Happy to switch it back, but I do think `expr` is probably the more accurate description for the macro right now (it's not actually a very good macro right now).